posts/tidy-mind-introduction.md

```diff
@@ -0,0 +1,110 @@
+# tidyMind - An introduction
+
+I use [Evernote][evernote] everyday, as my main note-taking app, as a place to put things off my mind. I like it a lot
+and have been using it for the last 4.5 years.
+
+But there are some things that I miss and think I could do better 🙃.
+
+Let's start with what I use Evernote for:
+
+On the personal side I have to do lists, a shopping list, notes about software projects, my server documentation,
+recipes (and some notes about how to cook food in general 😋), ideas for projects and blog posts, a bunch of (check)
+lists for packing, festivals, moving, skiing, things to check out on vacations, movies to watch, gift ideas, stuff I'd
+like to learn, and so on.
+
+At work I try to document most of the things in [Confluence][confluence] so it helps everybody 😇. But I have some notes
+about tickets I am working on (if I think about something that my current ticket I write it down immediately, so it's
+out of my mind and I don't forget it.)
+But I also keep there some early ideas which are not yet ready to be discussed in the team, a list of lunch options
+around the office and a journal where I write down what I worked on every day.
```

Comment:
do you really have a journal? that’s probably a good idea.

Reply:
I have indeed!